src/main/java/org/terasology/metalrenegades/ai/system/CitizenPersonalitySystem.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Please use the shorter license header 😉

Reply:
The license header has been changed 👍 The old license header still shows up for me when creating new files in IntelliJ, I have no idea why